sam-template.yaml

```diff
@@ -25,7 +25,7 @@ Resources:
             Description: Send scraper tasks to SQS
             Enabled: true
             Name: queue-builder
-            Schedule: rate(1 day)  # This could be 'cron ...': https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html
+            Schedule: rate(5 minutes)  # This could be 'cron ...': https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html
```

Comment:
I expect we should make some tooling to enqueue scrapers manually in AWS. An AWS "app" with some commands might be useful - e.g to list the queue length, recent logs, but also to manually write a job.